catalog/provider.go

```diff
@@ -32,6 +32,7 @@ type DatabaseProvider struct {
 	dsn                       string
 	externalProcedureRegistry sql.ExternalStoredProcedureRegistry
 	ready                     bool
+	initialDataDir            string
```

Comment:
This design complicates the deployment. Instead of requiring the presence of these CSV files, we can use Go's `embed` feature: https://pkg.go.dev/embed

Reply:
Thanks! I'll fix it in next commit.